packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java

```diff
@@ -107,6 +105,8 @@ public ResponseEntity<UUID> equationsToModel(@RequestBody final JsonNode req) {
 		} catch (final FeignException e) {
 			final String error = "Skema Unified Service did not return any AMR based on the provided Equations";
 			log.error(error, e);
+			if(e.status()<100)
```

Comment:
That is such a random number, do they return non HTTP error code sometimes?

Reply:
SKEMA seems to love returning `-1` as an HTTP status code which has blown things up before... anything above 100 is a valid code (though anything 100-299 would be unexpected???)